manifest/manifestlist/manifestlist_test.go

```diff
@@ -109,3 +110,105 @@ func TestManifestList(t *testing.T) {
 		}
 	}
 }
+
+// TODO (mikebrow): add annotations on the index and individual manifests
+var expectedOCIImageIndexSerialization = []byte(`{
```

Comment:
@owtaylor It would be easier just to use an OCI descriptor throughout distribution. They are the same.